tools/build_config/build.sh

```diff
@@ -12,7 +12,7 @@ readonly HORACE_ROOT="$(realpath $(dirname "$0")/../..)"
 # matlab executable. The Matlab on the path will likely be a symlink so we need
 # to resolve it with `readlink`
 readonly MATLAB_ROOT="$(realpath $(dirname $(readlink -f $(which matlab)))/..)"
-readonly MAX_CTEST_SUCCESS_OUTPUT_LENGTH=4096 #bytes
+readonly MAX_CTEST_SUCCESS_OUTPUT_LENGTH="10000" # 10 kilobytes
```

Comment:
10kb is 10240 bytes (using the same definition of KB as we were for 4k == 4096 kytes)

Reply:
Ignoring the 2^n vs 10^n debate; does this number need to be a string?

Reply:
I'm sure I changed it for a reason... But it seems to work as a number